src/main/java/dto/PersonDTO_OUT.java

```diff
@@ -11,19 +11,19 @@
 @Schema(name = "Person")
 public class PersonDTO_OUT {
 
-    @Schema(name = "ID", example = "1")
+    @Schema(description = "Unique Identifier", required = true, example = "1")
```

Comment:
I don't think the DTO_OUTs should have the required = true attribute. It doesn't make sense to say that something is required when it's not related to something the user is responsible for (that's what Lars said in the first video about API DOC)

Reply:
merge conflict error. will fix.